From 26856dcc0dcb66a56b5dfc0df0ae42c0155b367a Mon Sep 17 00:00:00 2001
From: Frank Earl <frank@earlconsult.com>
Date: Wed, 18 Sep 2019 22:13:27 -0500
Subject: Correct for build problems.

- Yes, we know that this is "wrong" but the codebase desperately
needs an enema, even past the clean state it's in.  It shouldn't
be stomping in time/date stamps in code in the FIRST place.

diff --git a/Makefile b/Makefile
index 10bec7a..5851005 100755
--- a/Makefile
+++ b/Makefile
@@ -16,10 +16,13 @@ EXTRA_CFLAGS += -Wno-unused-function
 EXTRA_CFLAGS += -Wno-unused
 #EXTRA_CFLAGS += -Wno-uninitialized
 
-GCC_VER_49 := $(shell echo `$(CC) -dumpversion | cut -f1-2 -d.` \>= 4.9 | bc )
-ifeq ($(GCC_VER_49),1)
-EXTRA_CFLAGS += -Wno-date-time	# Fix compile error && warning on gcc 4.9 and later
-endif
+#GCC_VER_49 := $(shell echo `$(CC) -dumpversion | cut -f1-2 -d.` \>= 4.9 | bc )
+#ifeq ($(GCC_VER_49),1)
+#EXTRA_CFLAGS += -Wno-date-time	# Fix compile error && warning on gcc 4.9 and later
+#endif
+
+# Turn OFF errors on date/time for GCC 7.x and later...
+EXTRA_CFLAGS += -Wno-error=date-time
 
 EXTRA_CFLAGS += -I$(src)/include
 
